english/light-fire/graham-crackers/chocolate/find-chocolate.md

```diff
@@ -1,11 +1,2 @@
 You don't find any chocolate in your jacket, but you do find some pre-fermented and dried cocoa beans and some sugar.
 
-You realize that you could make your own chocolate out of these ingredients,
-but it would probably take a while. You take a moment to weigh your options.
-
-You decide to:
-
-[Be lazy](../eat-goo/eat-goo.md) (just eat the graham crackers and marshmallow goo already)
-
-[Be patient](make-chocolate/make-chocolate.md) (try to make chocolate)
-
```

Comment:
Thank you for the help! I was trying to fix my mistake without starting over in a new branch & pull request, but it got confusing pretty quickly. I'll try out the `git cherry-pick` command you suggested.

Thanks for the great course. I am learning a lot!